dbptk-core/src/main/java/com/databasepreservation/modules/siard/out/output/SIARDDKDatabaseExportModule.java

```diff
@@ -0,0 +1,101 @@
+package com.databasepreservation.modules.siard.out.output;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Map;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.log4j.Logger;
+
+import com.databasepreservation.model.exception.ModuleException;
+import com.databasepreservation.modules.siard.common.path.MetadataPathStrategy;
+import com.databasepreservation.modules.siard.constants.SIARDDKConstants;
+import com.databasepreservation.modules.siard.out.metadata.ContextDocumentationWriter;
+import com.databasepreservation.modules.siard.out.metadata.FileIndexFileStrategy;
+import com.databasepreservation.modules.siard.out.metadata.SIARDMarshaller;
+
+/**
+ * @author Andreas Kring <andreas@magenta.dk>
+ *
+ */
+public class SIARDDKDatabaseExportModule extends SIARDExportDefault {
+
+  private SIARDDKExportModule siarddkExportModule;
+  private final Logger logger = Logger.getLogger(SIARDDKDatabaseExportModule.class);
+
+  public SIARDDKDatabaseExportModule(SIARDDKExportModule siarddkExportModule) {
+    super(siarddkExportModule.getContentExportStrategy(), siarddkExportModule.getMainContainer(), siarddkExportModule
+      .getWriteStrategy(), siarddkExportModule.getMetadataExportStrategy());
+
+    this.siarddkExportModule = siarddkExportModule;
+  }
+
+  @Override
+  public void initDatabase() throws ModuleException {
+    super.initDatabase();
+
+    // Delete output folder if it already exists
+
+    File outputFolder = siarddkExportModule.getMainContainer().getPath().toFile();
+    if (outputFolder.isDirectory()) {
+      try {
+        FileUtils.deleteDirectory(outputFolder);
+
+        // TO-DO: not logging ?
+
+        logger.info("Deleted the already existing folder: " + outputFolder);
+      } catch (IOException e) {
+        e.printStackTrace();
```

Comment:
Never commit a print stack trace. Also, in this situation if it failed deleting the directory it should send a ModuleException. Finally, it should not have this destructive behavior by default.